fdroid/repo/index.xml

```diff
@@ -49,7 +52,7 @@ Bitwarden is focused on open source software. The source code for Bitwarden is h
 			<size>71265987</size>
 			<sdkver>29</sdkver>
 			<targetSdkVersion>34</targetSdkVersion>
-			<added>2024-10-16</added>
```

Comment:
🤔 Now that we have multiple versions it's weird that this isn't using the github release date. I'm guessing it updated now because we changed the url / qrcode?

Reply:
I think this is generated by the F-Droid tools. It's using the `/diffs` files to determine the add date. We're not tracking all the diff files every time the F-Droid tools run so it thinks it is new every time.